src/main.rs

```diff
@@ -576,6 +579,16 @@ fn get_setup() -> Setup {
         ZEROCONF_INTERFACE,
         "Comma-separated interface IP addresses on which zeroconf will bind. Defaults to all interfaces. Ignored by DNS-SD.",
         "IP"
+    ).optopt(
+        "",
+        INTERPOLATION_QUALITY,
+        "Interpolation Quality to use if Resampling {Low|Medium|High}. Defaults to Low.",
```

Comment:
Not Medium? Kind of like the 160 kbps default? Seems saner.

Reply:
Let’s do it.

Reply:
Done.

Reply:
In hindsight, if CPU load is so low why provide a command line option at all?

Reply:
Librespot can technically be compiled for much slower devices. Or in the case maybe where a Pi Zero needs the CPU to do other things that a user may deem more important then librespot.

Reply:
In most cases though since it is dual threaded it shouldn't saturate a core on multi core devices. That was also a design choice. Technically it's ever so slightly more efficient for the resampler to be single threaded but then a single core would have to bear the whole weight of it. Basically dual threading it makes it more multi core friendly. It seemed like the right call especially because of how easily it's multi threaded since the different channels are processed independently anyway.